components/button-stateful/index.jsx

```diff
@@ -196,10 +199,18 @@ class ButtonStateful extends React.Component {
 
 		const isActive = typeof active === 'boolean' ? active : this.state.active;
 
+		// Accept aria-* props
+		let ariaProps = getAriaProps(this.props);
+
 		if (variant === 'icon' || variant === 'icon-filled') {
+			// Default aria attribute for stateful button with icon, if none is specified
+			if (!ariaProps || Object.keys(ariaProps).length === 0) {
```

Comment:
I think `!ariaProps` here could be removed because `getAriaProps` will always return an object.  The occurrence down below too?